yangify/translator/__init__.py

```diff
@@ -490,7 +504,16 @@ def __init__(
             o = None
 
         super().__init__(
-            None, None, instance.InstanceRoute(), dm, dm.schema, {}, n, o, replace
+            None,
```

Comment:
Oof this is not the greatest readability. Would you mind using keyword arguments for these to improve this? (lines 507-516)

Reply:
fixed